README.md

```diff
@@ -30,26 +30,42 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+The biggest difference between .forEach and .map is that .map creates a new array automatically while with .forEach you have to manually establish a new array.
```

Comment:
Your going in the correct direction.  `.map()` returns a new array, while `.forEach()` is just a for loop in which you can apply a `.push()` etc to create a new array.